src/test/java/org/agileandbeyond/dandd/tddexercise/CharacterTest.java

```diff
@@ -255,4 +255,33 @@ public void shouldHaveWeaponSetToTheSameWhenSet() throws Exception {
 		assertSame(weapon, dagger);
 	}
 	
+	@Test
```

Comment:
I would suggest creating another test in the character tests that instantiates an Orc and Human and then asserts that the armor class and abilities are modified properly.